test/phoenix_live_view/html_formatter_test.exs

```diff
@@ -87,6 +87,45 @@ if Version.match?(System.version(), ">= 1.13.0") do
       assert_formatter_doesnt_change(input)
     end
 
+    test "do not break between eex tags when there is no space before or after" do
```

Comment:
We need to the same for inline tags. I guess the current code doesn't cover:

```elixir
      assert_formatter_output(
        """
        <p>first <span>name</span>second</p>
        """,
        """
        <p>
          first <span>name</span>second
        </p>
        """,
        line_length: 20
      )
```

Reply:
> We need to the same for inline tags

You're right It doesn't cover inline tags, this PR is focused on eex tags only. I guess it's fine to add it in this PR, I'm gonna work on it.